packages/bridge-ui/src/components/ChainSelectors/ChainPill/ChainPill.svelte

```diff
@@ -9,10 +9,10 @@
   import { classNames } from '$libs/util/classNames';
   import { truncateString } from '$libs/util/truncateString';
   import { uid } from '$libs/util/uid';
+  import { account } from '$stores';
 
   export let value: Maybe<Chain> | null = null;
   export let label = '';
```

Comment:
```suggestion
  export let label = '';
  export let readOnly = false;
```

Even though `readOnly` is currently unused, please do not remove this functionality.